homeassistant/components/rointe/climate.py

```diff
@@ -0,0 +1,261 @@
+"""Support for Rointe Climate."""
+
+from __future__ import annotations
+
+from abc import ABC
+
+from rointesdk.device import RointeDevice
+
+from homeassistant.components.climate import (
+    PRESET_COMFORT,
+    PRESET_ECO,
+    ClimateEntity,
+    ClimateEntityDescription,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    CMD_SET_HVAC_MODE,
+    CMD_SET_PRESET,
+    CMD_SET_TEMP,
+    DOMAIN,
+    LOGGER,
+    PRESET_ROINTE_ICE,
+    RADIATOR_MODE_AUTO,
+    RADIATOR_MODE_MANUAL,
+    RADIATOR_PRESET_COMFORT,
+    RADIATOR_PRESET_ECO,
+    RADIATOR_PRESET_ICE,
+    RADIATOR_TEMP_MAX,
+    RADIATOR_TEMP_MIN,
+    RADIATOR_TEMP_STEP,
+)
+from .coordinator import RointeDataUpdateCoordinator
+from .rointe_entity import RointeRadiatorEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the radiator climate entity from the config entry."""
+    coordinator: RointeDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    # Register the Entity classes and platform on the coordinator.
+    coordinator.add_entities_for_seen_keys(
+        async_add_entities, [RointeHaClimate], "climate"
+    )
+
+
+class RointeHaClimate(RointeRadiatorEntity, ClimateEntity, ABC):
```

Comment:
`ClimateEntity` inherits from `Entity`, which inherits from `ABC`—is there a reason to inherit from `ABC` again here?